src/shogun/base/class_list.cpp.templ

```diff
@@ -74,3 +74,14 @@ void shogun::delete_object(CSGObject* object)
 {
 	delete object;
 }
+
+std::set<std::string> available_objects()
```

Comment:
is this exposed to swig?

Reply:
I am not sure if it works. But would work if we include std_set.i